tests/integration_test.rs

```diff
@@ -0,0 +1,239 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+use pretty_assertions::assert_eq;
+
+use wasm_dep_analyzer::Export;
+use wasm_dep_analyzer::ExportType;
+use wasm_dep_analyzer::GlobalType;
+use wasm_dep_analyzer::Import;
+use wasm_dep_analyzer::ImportType;
+use wasm_dep_analyzer::Limits;
+use wasm_dep_analyzer::TableType;
+use wasm_dep_analyzer::TagType;
+use wasm_dep_analyzer::WasmDeps;
+
+#[test]
+fn wasm_export_only() {
+  // The following Rust code compiled:
+  //
+  // #[no_mangle]
+  // pub fn add(left: usize, right: Usize) -> usize {
+  //     left + right
+  // }
+  let input = std::fs::read("tests/testdata/export_only.wasm").unwrap();
+  let module = WasmDeps::parse(&input).unwrap();
+  assert_eq!(
+    module,
+    WasmDeps {
+      imports: vec![],
+      exports: vec![
+        Export {
+          name: "memory",
+          index: 0,
+          export_type: ExportType::Memory,
+        },
+        Export {
+          name: "add",
+          index: 0,
+          export_type: ExportType::Function,
+        },
+        Export {
+          name: "__data_end",
+          index: 1,
+          export_type: ExportType::Global,
+        },
+        Export {
+          name: "__heap_base",
+          index: 2,
+          export_type: ExportType::Global,
+        }
+      ],
```

Comment:
I've left the output of this as low level for now, but we should probably make this higher level in the future. Additionally, we should probably parse out functions definitions so that we can create typescript types from it.